docs/decisions/app/0006-use-nextjs.md

```diff
@@ -0,0 +1,101 @@
+# Use Next.js for Template
```

Comment:
nit: trying to minimize references to "template" the templates, since once this decision gets copied over to a project it makes no sense anymore

there are exceptions things that are specific to developing the template itself, that don't get copied over to projects, but all the docs/decisions/app/* get copied over so this wouldn't be an exception

```suggestion
# Use Next.js for application framework
```